sdk/healthinsights/health-insights-cancerprofiling-rest/samples-dev/sample_infer_cancer_profiling.ts

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Infer key cancer attributes such as tumor site, histology, clinical stage TNM categories and pathologic stage TNM
+ *  categories from a patient's unstructured clinical documents.
+ *
+ * @summary detects change points.
+ */
+
+import { AzureKeyCredential } from "@azure/core-auth";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+import CancerProfilingRestClient, {
+    getLongRunningPoller,
+    isUnexpected, OncoPhenotypeData,
+    OncoPhenotypeResultOutput
+} from "../src";
+dotenv.config();
+
+// You will need to set this environment variables or edit the following values
+const endpoint = process.env["HEALTH_INSIGHTS_ENDPOINT"] || "https://eastus.api.cognitive.microsoft.com";
+const apiKey = process.env["HEALTH_INSIGHTS_API_KEY"] || "";
+
+function printResults(cancerProfilingResult: OncoPhenotypeResultOutput): void {
+    if (cancerProfilingResult.status === "succeeded") {
```

Comment:
can we use !isUnexpected instead?

Reply:
@joheredi I don't think isUnexpected could work here, isUnexpected would only check the status(aka HTTP status code) of the raw response. Here the status is the polling status located in `body` of the final response. 

@reutgross Generally RLC would not throw exceptions if the final response status code of polling is 200 when calling the `getLongRunningPoller` no matter what the polling status is, Succeeded or Failed. But if you only care about the `succeeded` result you could set the `resolveOnUnsuccessful`:

```typescript
try {
  const poller = await getLongRunningPoller(client, initialResponse, {resolveOnUnsuccessful: false});
  const cancerProfilingResult = await poller.pollUntilDone();
} catch (e) {
  // exceptions would be thrown when the polling status is not `succeeded`
}
```